android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -594,4 +595,11 @@ public String getApiName()
 	{
 		return "Ti.UI.Window";
 	}
+
+	private void logDeprecatedPropertiesWarning() {
```

Comment:
logDeprecatedPropertiesWarning();
 +			topActivity.startActivity(intent);
 +			int enterAnimation = TiConvert.toInt(options.get(TiC.PROPERTY_ACTIVITY_ENTER_ANIMATION), 0);
 +			int exitAnimation = TiConvert.toInt(options.get(TiC.PROPERTY_ACTIVITY_EXIT_ANIMATION), 0);
 +			topActivity.overridePendingTransition(enterAnimation, exitAnimation);


the convert methods use the old variables as they have not been changed to the new ones yet and so if the user uses the new ones it will fail wont it ?